python/ray/function_manager.py

```diff
@@ -158,36 +158,37 @@ def export(self, remote_function):
     def fetch_and_register_remote_function(self, key):
         """Import a remote function."""
         (job_id_str, function_id_str, function_name, serialized_function,
-         num_return_vals, module, resources,
```

Comment:
we don't store the `num_return_vals` and `resources` in redis. And we also never use it in the following code.